localstack/services/cloudformation/service_models.py

```diff
@@ -72,6 +72,9 @@ def update_resource(self, new_resource, stack_name, resources):
         # get_deploy_templates() responses, and based on checking whether resource attributes have changed
         pass
 
+    def is_updatable(self) -> bool:
+        return type(self).update_resource != GenericBaseModel.update_resource
```

Comment:
We could alternatively also raise a NotImplementedError and catch that in the deployer but this should work fine as well :thinking: